deno/gateway/v8.ts

```diff
@@ -30,7 +30,7 @@ export const GatewayVersion = '8';
 /**
  * https://discord.com/developers/docs/topics/opcodes-and-status-codes#gateway-gateway-opcodes
  */
-export enum GatewayOPCodes {
+export enum GatewayOpcodes {
```

Comment:
*OPCode* stands for *OP Code*, which is an abbreviation of *Operation Code*, as such, the enum's name should be OpCode:
```suggestion
export enum GatewayOpCodes {
```

Reply:
Also rename all references to it.